test/unit/redux/studio-member-actions.test.js

```diff
@@ -293,7 +293,7 @@ describe('inviteCurator', () => {
     });
     test('error because of rate limit', async () => {
         api.mockImplementation((opts, callback) => {
-            callback(null, null, {statusCode: 429});
+            callback(null, {}, {statusCode: 429});
```

Comment:
Could you explain why you needed this change?

Reply:
That field is the body of the response, which our actual responses all have. `normalizeError()` now checks `body.message` to distinguish between `RATE_LIMIT` (message is null) and `PASSWORD_ATTEMPT_LIMIT` (message is "try again later.") If the body is null, that check would error out.